packages/files-ui/src/Components/Modules/Settings/index.tsx

```diff
@@ -136,10 +128,9 @@ const useStyles = makeStyles(({ constants, breakpoints, palette }: ITheme) =>
 
 const Settings: React.FC = () => {
   const { desktop } = useThemeSwitcher()
-  const { path = desktop ? "profile" : undefined } = useParams<{path: SettingsPath}>()
+  const { path = desktop ? "profile" : "" } = useParams<{path: SettingsPath}>()
```

Comment:
This breaks the typing and obliged you to add a hack below. Any reason to change this?
```suggestion
  const { path = desktop ? "profile" : undefined } = useParams<{path: SettingsPath}>()
```

Reply:
Wasn't a deliberate change, the merges made it difficult to figure out what was a required change or not